client-v2/src/selectors/frontsSelectors.js

```diff
@@ -4,64 +4,68 @@ import { createSelector } from 'reselect';
 import { breakingNewsFrontId } from '../constants/fronts';
 
 import type {
-  FrontConfig,
-  FrontDetail,
-  FrontsClientConfig,
   Front,
-  ConfigCollection
+  ConfigCollection,
+  FrontDetailFull,
+  FrontsWithIds,
+  FrontsByPriority
 } from '../types/FrontsConfig';
 
 import type { State } from '../types/State';
 
-const frontsSelector = (state: State): Front => state.frontsConfig.fronts;
-const collectionsSelector = (state: State): ConfigCollection =>
-  state.frontsConfig.collections;
+const getFrontsConfig = (state: State) => state.frontsConfig || {};
 
-const prioritySelector = (state: State, priority: string) => priority;
+const rawFrontsSelector = (state: State): Front =>
+  getFrontsConfig(state).fronts || {};
 
-const frontsIdSelector = createSelector([frontsSelector], fronts => {
+const getFronts = createSelector(
+  [rawFrontsSelector],
+  (fronts: Front): FrontsWithIds =>
+    Object.keys(fronts)
+      .map((id): FrontDetailFull => ({
+        id,
+        ...fronts[id],
+        priority: fronts[id].priority || 'editorial'
```

Comment:
we should remove breaking news fronts here

Reply:
Yeah good plan, is that just a precaution? I need to quickly run through with you the way a breaking news front may appear not sure I quite understand it!

Reply:
The only thing to understand about the breaking news front is that we just don't want to show it here at all, it has it's on separate breaking news page which has slightly different behaviour from everything else. I saw you filtering the breaking news front further down this file but I don't think we should be adding it an editorial priority here as it is not an editorial front and does not get pressed.